arangod/Aql/RestAqlHandler.cpp

```diff
@@ -445,7 +445,7 @@ void RestAqlHandler::createQueryFromVelocyPack() {
 
   _qId = TRI_NewTickServer();
   try {
-    _queryRegistry->insert(_qId, query.get(), ttl, true);
+    _queryRegistry->insert(_qId, query.get(), ttl, true, false);
```

Comment:
Can this be changed so that `std::move(query)` is passed to QueryRegistry::insert instead of passing the raw pointer and then releasing it somewhere below?